.teamcity/components/settings.kt

```diff
@@ -138,7 +138,7 @@ var serviceTestConfigurationOverrides = mapOf(
         "network" to testConfiguration(timeout = 24),
 
         // New Relic is only available in East US region
```

Comment:
can we update the comment to specify **East US** is production region and **centraluseuap** is canary region and we are running tests on canary?